plugin/pkg/shoot/seedmanager/validation/validation.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validation
+
+import (
+	"fmt"
+	seedmanagerapi "github.com/gardener/gardener/plugin/pkg/shoot/seedmanager/apis/seedmanager"
+)
+
+// ValidateConfiguration validates the configuration.
+func ValidateConfiguration(config *seedmanagerapi.AdmissionConfigurationSeedmanager) error {
+
+	// will be set to default strategy by seedmanager plugin
```

Comment:
Either we default it in the API directly (then we have to make the field optional there) or we return an error here. Let's not default it where it is used.

Reply:
I would implement it in the way that:
- if there is no configuration provided (no flag): use DefaultStrategy/SameRegion
- if there is configuration provided, but the candidateStrategy field is empty or invalid: throw an error -> when there is an explicit configuration it has to be valid and we do not default it.

Does that make sense?

Reply:
Sounds good! :)